app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/Adapters/ExpandableListViewAdapter.java

```diff
@@ -0,0 +1,201 @@
+package ithaca_transit.android.cornellappdev.com.ithaca_transit.Adapters;
```

Comment:
this file was added, but the screenshots do not show any expandablelistviews? was this file meant to be pushed?